wiki/Installation/th.md

```diff
@@ -1,26 +1,68 @@
----
-outdated: true
----
+# วิธีการติดตั้ง
 
-# Installation (ไทย)
+ดูหน้านี้ด้วย: [การติดตั้งบน macOS](ใส่ลิงค์)
```

Comment:
```suggestion
ดูหน้านี้ด้วย: [การติดตั้งบน macOS](ใส่ลิงก์)
```

Reply:
I added a link from the English version instead.